sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -256,7 +289,8 @@ class SQLAppStatusListener(
     }
 
     val aggregatedMetrics = allMetrics.map { case (id, values) =>
-      id -> SQLMetrics.stringValue(metricTypes(id), values, maxMetricsFromAllStages.getOrElse(id,
+      val aggMethod = metricAggregationMethods(id)
+      id -> aggMethod(values, maxMetricsFromAllStages.getOrElse(id,
         Array.empty[Long]))
```

Comment:
nit: this can be merged to the line above.

Reply:
merged.